src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -612,26 +612,10 @@ public override string ToString()
         }
 
         public static bool operator ==(Color left, Color right)
-        {
-            if (left.value == right.value
+            => left.value == right.value
```

Comment:
Nit: can you move the => to the end of the line above?  That's where we've placed it elsewhere in the codebase.